### DISCUSSION TITLE
ROS2 packages for interfacing

### DISCUSSION THREAD

**Discussion by Akshat2400** [ros-industrial/industrial_core#294] [Q&A]
Is there any Repo which utilizes some of the ported ROS2 packages of ros_industrial_core to interface with staubli robots?

**Comment by gavanderhoorn** [comment]
I'm not sure I understand what you mean by that.

Could you clarify what you're looking for?

If you're referring to specific repositories or branches, please link to those so it's clear what the context is.

**Reply by Akshat2400** [reply]
I am working to develop a ros2 driver for staubli robots which require the ported industrial_core packages. I referred to ppbrown's repository but it is not fully ported.

**Reply by gavanderhoorn** [reply]
Does [IvoD1998/Staubli_ROS2](https://github.com/IvoD1998/Staubli_ROS2) not work for you?

And please *link* to repositories you are referring to. We don't necessarily know what *ppbrown's repository* is.
